services/order.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.models import User
+from django.db import transaction
+
+from db.models import Order
+
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The function does not handle the case where a user with the specified username does not exist. Consider using `get_object_or_404` or handling the `User.DoesNotExist` exception to provide a more user-friendly error handling.